msticpy/data/data_providers.py

```diff
@@ -53,7 +53,10 @@ class QueryProvider:
     """
 
     def __init__(
-        self, data_environment: Union[str, DataEnvironment], driver: DriverBase = None
+        self,
+        data_environment: Union[str, DataEnvironment],
+        driver: DriverBase = None,
+        query_path: str = path.join(path.dirname(__file__), _QUERY_DEF_DIR),
     ):
         """
         Query provider interface to queries.
```

Comment:
We should modify this and document using the custom config file to specify a folder with custom queries that are automatically loaded.

Reply:
Implemented